cartography/intel/aws/apigateway.py

```diff
@@ -107,222 +112,145 @@ def get_rest_api_policy(api: Dict, client: botocore.client.BaseClient) -> Any:
     return policy
```

Comment:
GH wont let me comment on line 107 but `get_rest_api_policy()` doesn't use the boto client param, can update function header